src/input-schemas.json

```
@@ -0,0 +1,1028 @@
+      "description": "The maximum amount of energy across the timeframe (e.g., a year) that the asset must produce. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    },
+    "milestone_year": {
+      "UoM": null,
+      "constraints": null,
+      "default": 2000,
+      "description": "Year of investment and operation decisions in the optimization.",
+      "type": "INTEGER"
+    },
+    "min_energy_timeframe_partition": {
+      "UoM": "MWh",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": null,
+      "description": "The minimum amount of energy across the timeframe (e.g., a year) that the asset must produce. If the initial value is null, empty, or missing, it will be no limit.",
+      "type": "DOUBLE"
+    },
+    "peak_demand": {
+      "UoM": "MW",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": 0,
+      "description": "Value that multiplies the demand profile time series.",
+      "type": "DOUBLE"
+    },
+    "storage_inflows": {
+      "UoM": "MWh/year",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": 0,
+      "description": "Value that multiplies the inflow profile time series.",
+      "type": "DOUBLE"
+    },
+    "units_on_cost": {
+      "UoM": "CUR/p.u./h",
+      "constraints": {
+        "minimum": 0
+      },
+      "deafult": 0,
+      "description": "Cost of keeping unit online for one hour or the objective function coefficient on `units_on` variable. e.g., no_load cost or idling cost",
+      "type": "DOUBLE"
+    }
+  },
+  "assets_profiles": {
+    "asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    },
+    "commission_year": {
+      "UoM": null,
+      "constraints": null,
+      "default": 2000,
+      "description": "Year of commissioning",
+      "type": "INTEGER"
+    },
+    "profile_name": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of profile, used to determine data inside the DuckDB table",
+      "type": "VARCHAR"
+    },
+    "profile_type": {
+      "UoM": null,
+      "constraints": {
+        "OneOf": [
+          "availability",
+          "demand",
+          "inflows",
+          "max_storage_level",
+          "min_storage_level",
+          "max_energy",
+          "min_energy"
+        ]
+      },
+      "default": null,
+      "description": "Type of profile, used to determine DuckDB table with source profile",
+      "type": "VARCHAR"
+    }
+  },
+  "assets_rep_periods_partitions": {
+    "asset": {
+      "UoM": null,
+      "constraints": null,
+      "default": null,
+      "description": "Name of the asset. Same as the one in the `asset` table.",
+      "type": "VARCHAR"
+    },
+    "partition": {
+      "UoM": null,
+      "constraints": null,
+      "default": 1,
```

```suggestion
      "default": "1",
```
At least I expect it to have a difference.

---

Why string?